flux/cluster/kustomization.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - upstream/crds.yaml
+  - upstream/cluster-roles.yaml
+  - upstream/cluster-role-bindings.yaml
+
+patchesJson6902:
+  - target:
```

Comment:
Not obvious to me why we need these patches

Reply:
otherwise there will be object in kube-system with general name `crd-controller` & `cluster-reconciler` used only by flux.
this patch will rename it to add `flux-` prefix.

Reply:
Could you add a comment to a similar effect so its also obvious in the code? :D :pray:

Reply:
done